server/handlers.go

```diff
@@ -36,6 +36,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"golang.org/x/net/idna"
```

Comment:
Lets move this down into the right section.

Reply:
hoped it was spotted by the linter :)

Reply:
I'll add a lint step for that 👍